packages/avatar-creator/src/scripts/avatar-loader.ts

```diff
@@ -392,6 +392,26 @@ export class AvatarLoader extends EventHandler {
     this.app.assets.load(asset);
   }
 
+  /**
+   * Get the avatar MML code for the current avatar
+   * @returns {string} the MML code for the current avatar
+   */
+  getAvatarMml() {
+    let code = "";
+    code += `<m-character src="${encodeURI(this.urls.torso ?? "")}">\n`;
```

Comment:
Any reason these can't be one line? i.e.
```suggestion
    let code = `<m-character src="${encodeURI(this.urls.torso ?? "")}">\n`;
```